Gemfile

```diff
@@ -60,6 +60,7 @@ if defined?(JRUBY_VERSION)
   gem 'jdbc-mariadb'
   gem 'jdbc-postgres'
   gem 'jdbc-sqlite3'
+  gem 'jdbc-mysql'
```

Comment:
NACK. As discussed, this will introduce a duplicate MySQL Driver on the classpath at runtime.

Original ticket: https://github.com/killbill/killbill-admin-ui-standalone/issues/12

Don't forget to use `jdbc:mariadb` in your jdbc connection string. See https://github.com/killbill/killbill-admin-ui-standalone/blob/b44d58c95019d4cee194b06fc439d0eb982cae08/config/database.yml#L11 for example.